packages/teleport/src/config.ts

```diff
@@ -61,9 +62,9 @@ const cfg = {
     player: '/web/cluster/:clusterId/session/:sid',
     sessionAuditPlayer: '/web/cluster/:clusterId/session/:sid/player',
     sessionAuditCmds: '/web/cluster/:clusterId/session/:sid/commands',
-    error: '/web/msg/error/:type?',
     login: '/web/login',
     loginFailed: '/web/msg/error/login_failed',
+    loginFailedCallback: '/web/msg/error/login_failed_callback',
```

Comment:
Instead, lets have a route

```
loginError: '/web/msg/error/login/',
loginErrorCallback: '/web/msg/error/login/call_back'
```

```tsx
 <Route path={cfg.routes.loginError} component={LoginError}  />
```

The `LoginError` will handle all it's children routes itself as well its own index route.

Reply:
revised